tasks/README.md

```diff
@@ -246,7 +246,7 @@ Future releases of the task runner may expand this or add configuration options
 
 ## Input Methods
 
-There are a three input methods available to tasks: [stdin](#stdin), [environment](#environment-variables), and [powershell](#powershell). By default tasks without a `.ps1` extension are passed parameters with both the `stdin` and `environment` input methods. Tasks with a `.ps1` extension are passed parameters with the `powershell` input method by default.
+There are four input methods available to tasks: [stdin](#stdin), [environment](#environment-variables), [positional](#positional) and [powershell](#powershell). By default tasks without a `.ps1` extension are passed parameters with both the `stdin` and `environment` input methods. Tasks with a `.ps1` extension are passed parameters with the `powershell` input method by default.  Positional parameters are passed to the task in the order specified by the parameter definition.
```

Comment:
The parameter metadata is in a JSON Object, and it is therefore not guaranteed that the read result is always in the order keys were written. With a Ruby runtime >= 1.9 the result is a hash with the keys in expected order, but this is a Ruby JSON implementation detail. To correctly specify the order I suggest either adding a `position` argument to a parameter definition, or to add a separate array with the parameter names in the wanted order.

Reply:
@hlindberg thanks - I have a position argument defined here: https://github.com/puppetlabs/puppet-specifications/pull/128/files#diff-f2efe900dbbc251d3e29d4e5fa32c701R104  Does that resolve your concern?   Or were you intending something else ?